src/epcpm/symtoproject.py

```diff
@@ -1202,7 +1202,6 @@ def parameter_from_signal(
             )
 
             if nv_meta is not None:
-                extras["nv_format"] = nv_meta.format
                 extras["nv_factor"] = nv_meta.factor
```

Comment:
`nv_factor` seems to have only possible value of POWER_SCALE. Do you know what that is and could we remove it?

Reply:
Seems like it has no effect unless `nv_format `is also specified. Same applies to `nv_cast`. Probably safe to remove both.